cclib/parser/gamessparser.py

```diff
@@ -1345,6 +1345,11 @@ def extract(self, inputfile, line):
                 for i in range(self.nbasis - base):  # Fewer lines each time
                     line = next(inputfile)
                     temp = line.split()
```

Comment:
Not your fault, but `temp` is a terrible name for any variable...can you give it a better name, just in the scope of the loop?  (I know `temp` is in a lot of other places in the code.)

The insertion is a nice trick but I think it's hard to understand without a better name.